projects/storefrontlib/src/lib/occ/user/user.service.ts

```diff
@@ -60,6 +60,41 @@ export class OccUserService {
       .pipe(catchError((error: any) => throwError(error)));
   }
 
+  addUserAddress(userId, address) {
+    const url = this.getUserEndpoint() + userId + ADDRESSES_ENDPOINT;
+    const headers = new HttpHeaders({
+      'Content-Type': 'application/json'
+    });
+
+    return this.http
+      .post(url, address, { headers })
+      .pipe(catchError((error: any) => throwError(error)));
+  }
+
+  updateUserAddress(userId, addressId, address) {
+    const url =
+      this.getUserEndpoint() + userId + ADDRESSES_ENDPOINT + '/' + addressId;
+    const headers = new HttpHeaders({
+      'Content-Type': 'application/json'
+    });
+
+    return this.http
+      .patch(url, address, { headers })
+      .pipe(catchError((error: any) => throwError(error)));
+  }
+
+  deleteUserAddress(userId, addressId) {
```

Comment:
Wouldn't be good to use TS for function params? Also for the functions below

Reply:
Params added.